tests/Constraints/MinMaxPropertiesTest.php

```diff
@@ -82,8 +82,26 @@ public function getInvalidTests()
                 }'
             ),
             array(
+                '{}',
+                '{
+                  "type": "object",
+                  "properties": {
+                    "propertyOne": {
+                      "type": "string"
+                    },
+                    "propertyTwo": {
+                      "type": "string"
+                    }
+                  },
+                  "minProperties": 1
+                }'
+            ),
+            array(
                 '{
-                  "value": 1
```

Comment:
because this test was incorrect, the object was found invalid for the wrong reason (because `value` was an integer instead of an object)